### PR TITLE
doc: remove SLE (not tested), change Leap 15 to Leap 15.0

### PR DESCRIPTION
**Pull request by csymons-suse**
remove reference to repositories not packaged

### CODE REVIEW

File: doc/source/deployment/requirements.rst [hunk 1]
```diff
@@ -10,11 +10,11 @@ requirements.
 Infrastructure
 --------------
 
-* The `Deployer` must run openSUSE Leap 15 or SUSE Linux Enterprise 15. See
-  :ref:`setupdeployer` for required deployment tools and packages.
+* The `Deployer` must run openSUSE Leap 15.0 (**Neither Leap 15 nor Leap 15.1
+  will work**). See :ref:`setupdeployer` for required deployment tools and packages.
```
Comment (jgu17) [inline doc/source/deployment/requirements.rst]: not sure about "neither ..." statement, it si correct, but sounds negative :-)

File: doc/source/deployment/setup-deployer.rst [hunk 2]
```diff
@@ -59,22 +59,15 @@ Deployer. Set up your workspace with the following steps:
...
 1. (Recommended) Install with an ISO image including required dependencies:
 
-   a. Download ``openSUSE-Addon-socok8s-x86_64-Media.iso`` from
-      https://download.opensuse.org/repositories/Cloud:/socok8s:/master/images/iso/
-   b. sudo zypper addrepo --refresh <PATH_TO_ISO_IMAGE> socok8s-iso
+   a. wget https://download.opensuse.org/repositories/Cloud:/socok8s/images/iso/openSUSE-Addon-socok8s-x86_64-Media.iso
+   b. sudo zypper addrepo --refresh iso:///?iso=/home/stack/openSUSE-Addon-socok8s-x86_64-Media.iso socok8s-iso
    c. sudo zypper install socok8s (installs to /usr/share/socok8s)
 
-2. Install from the openSUSE repository including required dependencies:
```
Comment (jgu17) [inline doc/source/deployment/setup-deployer.rst]: why taking out the rpm package install approach from the master branch?

File: doc/source/deployment/requirements.rst (hunk 1 above)
Comment (csymons-suse) [reply doc/source/deployment/requirements.rst]: changed in 35a8815

File: doc/source/deployment/setup-deployer.rst (hunk 2 above)
Comment (csymons-suse) [reply doc/source/deployment/setup-deployer.rst]: Should this alternative be available in master branch? It was removed previously and is not available in stable/1.0.

**Comment by dirkmueller** [comment]
we switched to SLE now, so this change no longer makes sense, sorry.

**Pull request closed without merging**